lab_6_pipeline/pipeline.py

```diff
@@ -207,6 +332,17 @@ def __init__(
             analyzer (LibraryWrapper): Analyzer instance
             pos (tuple[str, ...]): Root, Dependency, Child part of speech
         """
+        self._corpus = corpus_manager
+        self._analyzer = analyzer
+        self._node_labels = pos
+
+        self.ideal_graph = DiGraph()
+        self.ideal_graph.add_nodes_from(
+            (index, {'label': label})
+            for index, label in enumerate(self._node_labels)
+        )
+        self.ideal_graph.add_edges_from((index, index + 1)
+                                        for index in range(len(self._node_labels) - 1))
```

Comment:
it is ok for now. think how you would handle a case of finding a tree by pattern:

```
A
|   \
B     C

```

Reply:
I guess, it depends on the way we give this tree to the Pipeline. If we have the edges, then we use them, if nested dict/tuple, then recursively collect all edges

Reply:
I guess, it depends on the way we give this tree to the Pipeline. If we have the edges, then we use them, if nested dict/tuple, then recursively collect all edges

Reply:
interesting, demonstrate please how it will work